cocos/core/data/object.ts

```diff
@@ -266,7 +266,7 @@ class CCObject {
      * obj.destroy();
      * ```
      */
-    public destroy (): boolean {
+    public destroy (): boolean | void {
```

Comment:
Why add void here ? It should return true or false, no exception, all sub class destroy should also call `return super.destroy()` to see whether it's been successfully destroyed or not, it's also a chance to check and correct things

Reply:
That's because currently there are many derived classes return nothing